docs/performance/http-cache.md

```diff
@@ -0,0 +1,64 @@
+---
+title: HTTP Cache
+---
+HTTP Cache
+==========
+
+Bolt comes with [Symfony HTTP Cache][] that will act as a simple reverse proxy
+to cache rendered pages on disk for a period of time.
+
+This can provide a performance boost on some pages, as means less work for the
+werb server to do in order to serve the requested page repeatedly in a short
+amount of time.
+
+<p class="note"><strong>Note:</strong> Requests for logged on users will not
+be cached.
+</p>
+
+
+Enabling
+--------
+
+To enable [Symfony HTTP Cache][] for your Bolt install, in your `config.yml`
+file, find the `caching:` key, and enableing the `request` parameter.
+
+
+```
+caching:
+    request: true
+```
+
+
+Fine Tuning Options
```

Comment:
I think this should be [Fine-tuning](https://en.wikipedia.org/wiki/Fine-tuning), right?